src/genie/libs/parser/iosxe/show_cts.py

```diff
@@ -147,5 +147,305 @@ def filter_lines(raw_output, remove_lines):
                 refresh_timer = groups['refresh_timer']
                 cts_pacs_dict['refresh_timer'] = refresh_timer
                 continue
+
         return cts_pacs_dict
 
+
+
+# ==============================
+# Schema for:
+#  * 'show cts environment-data'
+# ==============================
+class ShowCtsEnvironmentDataSchema(MetaParser):
+    """Schema for show cts environment-data."""
+
+    schema = {
+        "cts_env": {
+            "current_state": str,
+            "last_status": str,
+            Optional("sgt_tags"): str,
+          Optional("tag_status"): str,
+          Optional("server_list_name"): str,
+          Optional("server_count"): int,
+          Optional("servers"): {
+              Optional(int): {
+                  Optional("server_ip"): str,
+                  Optional("port"): int,
+                  Optional("aid"): str,
+                  Optional("server_status"): str,
+                  Optional("auto_test"): str,
+                  Optional("keywrap_enable"): str,
+                  Optional("idle_time_mins"): int,
+                  Optional("dead_time_secs"): int
+              }
+          },
+          Optional("security_groups"): {
+              Optional(int): {
+                  Optional("sec_group"): str,
+                  Optional("sec_group_name"): str
+              }
+          },
+          Optional("env_data_lifetime_secs"): str,
+          Optional("last_update"): {
+            Optional("date"): str,
+            Optional("time"): str,
+            Optional("time_zone"): str
+          },
+          Optional("expiration"): str,
+          Optional("refresh"): str,
+          "state_machine_status": str,
+          Optional("retry_timer_status"): str
+        }
+    }
+
+
+
+# ==============================
+# Parser for:
+#  * 'show cts environment-data'
+# ==============================
+class ShowCtsEnvironmentData(ShowCtsEnvironmentDataSchema):
+    """Parser for show cts environment-data"""
+
+    cli_command = ['show cts environment-data']
+
+    def cli(self, output=None):
+        if output is None:
+            out = self.device.execute(self.cli_command[0])
```

Comment:
when you adjust above, no need to index cli_command

Reply:
This has been updated